romana-install/roles/romana/postinstall/kubernetes/master/templates/demo.sh

```diff
@@ -100,4 +107,4 @@ desc "this permits us to connect from frontend to backend"
 run "kubectl --namespace=tenant-a exec nginx-frontend -- curl $(get_pod_ip 'nginx-backend' 'tenant-a') --connect-timeout 5"
 
 desc "Demo completed (cleaning up)"
-run "curl -X DELETE http://{{ romana_master_ip }}:8080/apis/romana.io/demo/v1/namespaces/tenant-a/networkpolicys/pol1; kubectl --namespace=tenant-a delete pod nginx-backend; kubectl --namespace=tenant-a delete pod nginx-frontend; kubectl delete namespace tenant-a; kubectl delete replicationcontroller nginx-default"
+run "curl -X DELETE http://{{ romana_master_ip }}:8080/apis/romana.io/demo/v1/namespaces/tenant-a/networkpolicys/pol1; kubectl --namespace=tenant-a delete pod nginx-backend; kubectl --namespace=tenant-a delete pod nginx-frontend; kubectl delete namespace tenant-a; kubectl delete replicationcontroller nginx-default; $(delete_tenant 'tenant-a')"
```

Comment:
The last bit should _not_ be wrapped in `$(...)`.
It makes it run too early (when evaluating the line itself), and then the `run` function will try to execute the output of it. Which is luckily empty string.